rpc/jsonrpc/types/chainsvrresults.go

```diff
@@ -62,6 +62,7 @@ type GetAddedNodeInfoResult struct {
 // hex-encoded string.  Contains Decred additions.
 type GetBlockVerboseResult struct {
 	Hash          string        `json:"hash"`
+	PoWHash       string        `json:"powhash"`
```

Comment:
Please make the two changes and update the file's copyright year in a separate commit:

`jsonrpc/types: Add powhash to verbose block output.` 

Since `types` is a separate module, doing that will allow me to tag a new version with the new field right away without having to do more commits.

It's probably obvious, but the new commit would need to come before the main `rpc` commit.